src/tests/account/test_eth_account.cairo

```diff
@@ -24,7 +25,6 @@ use poseidon::poseidon_hash_span;
 use starknet::ContractAddress;
 use starknet::SyscallResultTrait;
 use starknet::account::Call;
-use starknet::eth_signature::Signature;
```

Comment:
There's also an unused `eth_signature::Signature` in [test_dual_eth_account](https://github.com/ericnordelo/cairo-contracts/blob/feat/eth_account_improvement/src/tests/account/test_dual_eth_account.cairo#L15) that we can demolish 💣